pkg/util/kube.go

```diff
@@ -13,4 +13,8 @@ const (
 	VolumeGIDAnnotation = "volume.podman.io/gid"
 	// Kube annotation for podman volume mount options.
 	VolumeMountOptsAnnotation = "volume.podman.io/mount-options"
+	// Kube annotation to set registry for auto-update policies in containers.
+	AutoUpdateRegistryAnnotation = "io.containers.autoupdate"
+	// Kube annotation to set registry authfile for auto-update policies in containers.
+	AutoUpdateAuthfileAnnotation = "io.containers.autoupdate.authfile"
```

Comment:
I think we should use the exported variables in `pkg/autoupdate` and not introduce copies.

Reply:
I have made the requested changes. For `podman play`, the const vars will be obtained from `pkg/autoupdate`. For generate, the file `podman/libpod/kube.go` will use a var containing `"io.containers.autoupdate"` since `pkg/autoupdate` is already importing `libpod` and the const vars couldn't be imported due to `import cycle not allowed` error.

Maybe the logic in `podman/libpod/kube.go` could be taken out to an external  pkg, similar to `pkg/systemd`.